packages/ssx-sdk/src/modules/Encryption.ts

```diff
@@ -51,14 +52,224 @@ interface IEncryption {
   decrypt: (encrypted: any) => Promise<any>;
 }
 
+/**
+ * The LitEncryptData interface describes the parameters for the 
+ * LitEncryption's encrypt method.
+ */
+interface LitEncryptData {
+  /**
+   * The data to encrypt.
+   */
+  content: string;
+  /** 
+   * Lit's access control. Check out 
+   * {@link https://developer.litprotocol.com/accessControl/intro Lit's docs} 
+   * for more information. 
+   */
+  accessControlConditions: Array<any>;
+}
+
+/**
+ * The LitDencryptData interface describes the parameters for the 
+ * LitEncryption's decrypt method.
+ */
+interface LitDecryptData {
+  /** Blob content returned by the encrypt method. */
+  encryptedString: Blob;
+  /** encryptedSymmetricKey content returned by the encrypt method. */
+  encryptedSymmetricKey: string;
+  /**
+   * Lit's access control. Check out 
+   * {@link https://developer.litprotocol.com/accessControl/intro Lit's docs} 
+   * for more information. 
+   */
+  accessControlConditions: Array<any>;
+}
+
+/**
+ * The LitEncryption module handles the encryption and decryption of data.
+ * It uses the user's signature (via UserAuthorization) and access control 
+ * conditions to encrypt data and allow decrypt if the access conditions
+ * are satisfied.
+ */
 class LitEncryption implements IEncryption {
   private userAuth: IUserAuthorization;
+  private client = new LitJsSdk.LitNodeClient({});
+  private litNodeClient;
 
   constructor(config: any, userAuth: IUserAuthorization) {
     this.userAuth = userAuth;
   }
-  encrypt: (data: any) => Promise<any>;
-  decrypt: (encrypted: any) => Promise<any>;
+
+  /**
+   * Given the chainId, returns the chain name following the supported chains. Check out 
+   * {@link https://developer.litprotocol.com/resources/supportedChains Lit's docs} 
+   * for more information. 
+   * @param chainId - Unique chain id.
+   * @returns - The chain name follwing the Lit Protocol docs.
+   */
+  private getChainName = (chainId: number) => {
```

Comment:
Had to create this to translate chainId -> chain string following[Lit supported chains](https://developer.litprotocol.com/resources/supportedchains/).